extras/docs/troubleshooting.rst

```diff
@@ -0,0 +1,86 @@
+Troubleshooting Guide
+=====================
+
+First check : mod installation and configuration
+------------------------------------------------
+
+Go to your *KSP/GameData* folder. You should see a *KerbalSimpit* folder if SimPit is installed (else, install it with CKAN).
+
+Open the configuration file *KSP/GameData/KerbalSimpit/PluginData/Settings.cfg*
+
+The line ``PortName = COM3`` should be updated with the correct port number. It should be identical to the one used with the Arduino IDE.
+
+The line ``Verbose = False`` should be replaced by ``Verbose = True`` to increase the level of detail written in the logs. It is assumed for this guide that Verbose is set to True.
+
+Launch KSP and check in the logs that you can see the following lines (either by opening the in-game console with Alt+F12 or by opening the file *KSP/KSP.log* :
+ * ``KerbalSimpit Has put a message into the console!``
+ * ``KerbalSimpit: Settings loaded.``
+ * ``KerbalSimpit: Using serial polling thread for XXX`` where XXX is your port name as used in the Arduino IDE.
```

Comment:
On a Mac, it will list the port as Serial port 0 (which will not match what you see in the IDE or what was configured in the settings.cfg file

Reply:
Fixed on the next commit. Thanks